### ISSUE TITLE
Sensing and Pseudo-Sensing of obstacles

### ISSUE THREAD

**Issue by maxspahn** [maxspahn/gym_envs_planar#31]
Currently, obstacles (static and dynamic) and goals (static and dynamic) can be added to the scene. 
This is very helpful in visualization. However, it would be even better if the information could be 'sensed' and potentially passed to the observation if requested. 

The idea:
The code should be something like:
```python
sensor = PseudoSensor(obstacles=True, goals=True)
env.addSensor(sensor)
```
In the simulation loop the observation returned by the stepper should then include the information of obstacles and goals at that very time step.

A sensor class has already been added in the urdf environment: https://github.com/maxspahn/gym_envs_urdf/tree/master/sensors.
For this toy-like environment, a pseudo sensor should be enough to test motion planning.

Contact me if you have any questions on it.

**Comment by alxschwrz** [comment]
Hi Max,
I'll have a look at it and would like to implement the PseudoSensor for the environments.

**Comment by alxschwrz** [comment]
I am working on it on https://github.com/alxschwrz/gym_envs_planar/tree/ft-pseudosensor

**Comment by maxspahn** [comment]
Closed as #33 was accepted.
